client/controllers/userEvents.coffee

```diff
@@ -16,7 +16,7 @@ Template.userEvents.onCreated ->
       defaultSortDirection: -1,
       displayFn: (value, object, key) ->
         if value != null
-          return value.toLocaleString()
+          return moment(value.toLocaleString()).format('MMM D, YYYY')
```

Comment:
I think the toLocaleString calls can be removed from this and the following displayFns.